io.catenax.individual_asset_definition/1.0.0/IndividualAssetDefinition.ttl

```diff
@@ -0,0 +1,79 @@
+#######################################################################
+# Copyright (c) 2023 BASF SE
+# Copyright (c) 2023 Henkel AG & Co. KGaA
+# Copyright (c) 2023 ZF Friedrichshafen AG
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:2.0.0#>.
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:2.0.0#>.
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:2.0.0#>.
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:2.0.0#>.
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix : <urn:bamm:io.catenax.individual_asset_definition:1.0.0#>.
+@prefix ext-dimension: <urn:bamm:io.catenax.shared.physical_dimension:1.0.0#>.
+
+:IndividualAssetDefinition a bamm:Aspect;
+    bamm:preferredName "Individual Asset Definition"@en;
+    bamm:description "Permanent characteristics of an individual asset."@en;
+    bamm:properties (:catenaXId :assetOwnerId :assetSerialNumber :assetType ext-dimension:physicalDimensionsProperty);
+    bamm:operations ();
+    bamm:events ().
+:catenaXId a bamm:Property;
+    bamm:preferredName "Catena-X Identifier"@en;
+    bamm:description "The fully anonymous Catena-X ID of the asset, valid for the Catena-X dataspace."@en;
+    bamm:characteristic :CatenaXIdTrait;
+    bamm:exampleValue "urn:uuid:ed85f17e-29dd-473c-9cb8-d7ad1dc44d2f".
+:assetOwnerId a bamm:Property;
+    bamm:preferredName "Asset Owner ID"@en;
+    bamm:description "BPNL of the asset owner."@en;
+    bamm:characteristic :AssetOwnerCharacteristic;
+    bamm:exampleValue "BPNL021738173000".
+:assetSerialNumber a bamm:Property;
+    bamm:preferredName "Asset Serial Number"@en;
+    bamm:description "The serial number of the asset."@en;
+    bamm:characteristic :AssetSerialNumberCharacteristic;
+    bamm:exampleValue "321-0740-6790-XY".
+:assetType a bamm:Property;
+    bamm:preferredName "Asset Type"@en;
+    bamm:description "The type of an asset."@en;
+    bamm:characteristic :AssetTypeCharacteristic;
+    bamm:exampleValue "package".
+:CatenaXIdTrait a bamm-c:Trait;
+    bamm:preferredName "Catena-X Id Trait"@en;
+    bamm:description "Trait to ensure UUID v4 data format."@en;
+    bamm-c:baseCharacteristic :UUIDv4;
+    bamm-c:constraint :UUIDv4RegularExpression.
+:AssetOwnerCharacteristic a bamm:Characteristic;
```

Comment:
I always use :**string** for short text input and I use **:text** for longer expected input such as descriptions, comment content, etc.
In this case BPNL Number, Serial Number and AssetType are short text input.

Reply:
@Cvetanka-ZF I think @catroest refers to the used characteristic. In case there is only a generic text without a specific semantics (ie specific description attached to it) you can use the bamm-c:Text characteristic and not obliged to define one on your own.